### ISSUE TITLE
Build issues with latest master branch (as on 31st July, 2017)

### ISSUE THREAD

**Issue by genuinelucifer** [mono/CppSharp#905]
##### Brief Description
I built the required llvm and clang commits. Then I tried to generate project files for VS 2015 and found that there is a typo at https://github.com/mono/CppSharp/blob/master/build/LLVM.lua#L117 in the word 'action'. I fixed it and generated the projects.  
But in the generated solution, I am getting the following error when trying to build CppSharp:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2039	'classof': is not a member of 'CppSharp::CppParser::AST::DeclarationContext'	CppSharp.CppParser	d:\repos\cppsharp\deps\llvm\include\llvm\support\casting.h	59	
```
I double checked that I am using the correct commit for LLVM and clang. And can't think of a way to fix this..  
  
OS: Windows

##### Used headers
none

##### Used settings
default

Target: MSVC

**Comment by ddobrev** [comment]
Hello @genuinelucifer . I am happy to see you.
I assume you've built Clang yourself. You don't need to do it any more as we build and upload the required packages ourselves. Just follow the instructions at https://github.com/mono/CppSharp/blob/master/docs/GettingStarted.md#downloading-the-llvm-and-clang-packages and then https://github.com/mono/CppSharp/blob/master/docs/GettingStarted.md#compiling-on-windowsvisual-studio .

**Comment by genuinelucifer** [comment]
Oh great! I didn't know about that. I will try to use those packages and report back.

**Comment by genuinelucifer** [comment]
Closing as typo has been fixed in #906  and the error seems to occur because I was building in debug mode. Tried Release in x86 and it worked!
